common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -3898,6 +3898,9 @@ schematic = (function() {
 	    tool.style.verticalAlign = 'middle';
 	    tool.style.cursor = 'default';
 
+        // add alt using tip
+        tool.setAttribute('alt', tip);
+
```

Comment:
ugh. Just opened this file on my computer to check it out.... Looks like it's using mixed modes of indentation (tabs some places and spaces in others).  You followed the JS guidelines here, but unfortunately the rest of this block is using "tab space space space space" for indentation.  :-(   Any chance you can add another commit (without squashing) to this PR to make the indentation in this file consistent (e.g. all spaces, no tab characters)?  (I can show you an easy way to do it in sublime if it helps.)